src/ResourceManager/Sql/Commands.Sql/Failover Group/Cmdlet/NewAzureSqlFailoverGroup.cs

```diff
@@ -66,7 +67,7 @@ public class NewAzureSqlFailoverGroup : AzureSqlFailoverGroupCmdletBase
         [Parameter(Mandatory = false,
             HelpMessage = "The window of grace period that we tolerate with data loss during a failover operation for the failover group.")]
         [ValidateNotNullOrEmpty]
-        public int GracePeriodWithDataLossHour { get; set; }
+        public int GracePeriodWithDataLossHours { get; set; }
```

Comment:
@ziwa-msft this is again a breaking change. You can add a alias and add a deprecation warning that this parameter will be fixed in the next breaking change cycle.
We cannot ship any breaking changes